protecode/util.py

```diff
@@ -106,23 +154,69 @@ def iter_artefact_metadata(
                 data=license_finding,
                 discovery_date=discovery_date,
             )
-            has_license_findings = True
 
-        meta = dso.model.Metadata(
-            datasource=dso.model.Datasource.BDBA,
-            type=dso.model.Datatype.VULNERABILITY,
-            creation_date=now,
-            last_update=now,
-        )
+            # remove license finding because it is still relevant and must not be deleted afterwards
+            existing_findings = [
```

Comment:
After filtering this name (`existing_findings`) is not really accurate anymore.
How about some `stale_findings` ?